src/wp-includes/class-wp-theme.php

```diff
@@ -419,11 +439,12 @@ public function __construct( $theme_dir, $theme_root, $_child = null ) {
 				$this->cache_add(
 					'theme',
 					array(
-						'block_theme' => $this->is_block_theme(),
-						'headers'     => $this->headers,
-						'errors'      => $this->errors,
-						'stylesheet'  => $this->stylesheet,
-						'template'    => $this->template,
+						'block_theme'            => $this->is_block_theme(),
+						'headers'                => $this->headers,
+						'errors'                 => $this->errors,
+						'stylesheet'             => $this->stylesheet,
+						'template'               => $this->template,
+						'block_template_folders' => $this->block_theme ? $this->get_block_template_folders() : null,
```

Comment:
Can you explain this one? What about hybrid themes?

Reply:
Sorry, accidentally left behind a residual of my early code.  I have fixed it now.